chapters/en/slides/module2/module2_02.Rmd

```diff
@@ -0,0 +1,279 @@
+```
+
+Notes:
+Thanks to Altair's consistent visualization grammar,
+the modification we need to make to the code are minor.
+Replace `mark_line` with `mark_area` and we're done.
+
+For showing a singe trend over time,
+the choice between a line and area chart comes down to asthetics.
+They are both effective choices for communicating
+
+However,
+when visualizing the trends over time for multiple groups,
+lines and areas have different advantages,
+as we will see next.
+
+---
+
+## Area charts are preferred when the total of the groups is the most important
+
+```{python}
+alt.Chart(gm).mark_area().encode(
+    x='year',
+    y='sum(population)',
+    color='region')
+```
+
+Notes:
+The y-axis range of this chart
+is the same as the one on the previous slide,
+so it is easy to see how the total world population has changed over time.
+
+From the stacked colored regions,
+we also get a rough idea of how each region has grown,
+but it is hard to compare exactly,
+especially for regions that are not stacked next to each other.
+For example,
+we can't really tell if Europe, Africa, or the Americas
+has the largest population in the most recent year.
+
+---
+
+## Line charts are preferred when the values of the individual groups are the most important
+
+```{python}
+alt.Chart(gm).mark_line().encode(
+    x='year',
+    y='sum(population)',
+    color='region')
+```
+
+Notes:
+Using a line chart
+is the preferred choice when we want to view the exact values of each group
+and don't care as much about the total of the groups added together.
+
+For example,
+here it is immediately clear that Africa has a bigger population 
+than Europe and the America,
+and we can see that the populations shifted around year 2000.
+
+On the other hand,
+it's quite cognitively demanding
+to try to reconstruct the total world population
+by adding all the lines up together,
+especially over time!
+
+---
+
+## When will the world's population stop growing?
+
+```{python}
+alt.Chart(gm).mark_area().encode(
+    x='year',
+    y='sum(population)')
+```
+
+Notes:
+Going back to the area chart with one group,
+let's look closer at the growth curve of the world's population.
+
+First,
+there is a slow linear phase,
+but soon after the end of the second world war,
+it the growth starts increasing much more rapidly
+(but still in a straight line).
+
+Will the world population keep growing like this for much longer?
+
+---
+
+## The world population is predicted to stabilize around 11 billion
+
+```{python}
+alt.Chart(gm).mark_area().encode(
+    x='year',
+    y='mean(children_per_woman)')
+```
+
+Notes:
+Fortunately,
+the world population is predicted
+to stabilize around 11 billion people at the end of the century.
+
+Why?
+Because as living conditions improve around the world,
+child mortality is going down,
+which means that the number of children per women is decreasing
+towards a level than means that the current population will be sustained
+rather than increase.
```

Comment:
This is written as fact, where it is actually a researchers hypothesis. We should make sure that is clear. 

Also, this sentence is a run on does not read right: "Because as living conditions improve around the world,
child mortality is going down, which means that the number of children per women is decreasing towards a level than means that the current population will be sustained rather than increase."

Decreases in child mortality is not causing the number of children per women to decrease...

Reply:
My understanding is that decreasing child mortality is one of the main drivers in reducing the number of children per woman. This is presented as a causative relationship by both gapminder and [Our world in data](https://ourworldindata.org/fertility-rate#increasing-well-being-and-status-of-children) with several academic references, and I presented it the same way since these organizations are global health experts.

There are of course other factors that contribute to reduced fertility rates, and this relationship is not causative in the same strong sense as in natural sciences ([for example not in Niger](https://www-nature-com.myaccess.library.utoronto.ca/articles/542414a)), but this is a limitation of any observational study (so most global health science). I can rephrase to reflect this, but I still the most accurate to portray this as the literature in the field does, maybe something like:

"In countries where child mortality is high, women will often give birth to more children than they want in order to protect against losing some of their children prematurely. Lower child mortality leads to more predictable family planning and as child mortality goes down, the number of children per woman approaches 2 in the majority of countries. "